src/components/trade/PriceSuggestions/PriceSuggestionItem.tsx

```diff
@@ -28,18 +29,39 @@ interface FormattedPrices {
 
 const LOW_PRICE_FLOOR = new BigNumber('0.0001')
 
-export function formatPriceToPrecision(price: BigNumber, useThreshold = false): string {
-  return price.gt(LOW_PRICE_FLOOR) || !useThreshold
-    ? price.toFixed(PRICE_ESTIMATION_PRECISION)
-    : '< ' + LOW_PRICE_FLOOR.toString()
+function formatPriceWithFloor(price: BigNumber): string {
+  const priceAsBN = parseAmount(price.toString(10), 18)
```

Comment:
convertions utils in utils, pls

Reply:
removed conversion in favour of using big number built in conversion

Reply:
done